src/modelAnalysis/rumba/rumba.m

```diff
@@ -0,0 +1,294 @@
+function [RUMBA_outputs, UpRegulated, DownRegulated, MetConnectivity1, MetConnectivity2] = rumba(model1, model2, completeModel, sampling, maxMetConn, RxnsOfInterest, GenesOfInterest, NormalizePointsOption, PValCuttoff, MaxNumPoints, LoopRxnsToIgnore, verboseTag)
+% RUMBA predicts which reactions significantly change their

```

Comment:
Hi,
I implemented a inputParser for the method on my rumba branch ( commits: 771ff81 and  94917ea ), which could be cherry-picked onto your branch.
BUT: since your current branch is behind develop (i.e. not rebased recently), There is an issue with verifyModel on your current branch. I use the syntax of the updated one on the development branch, but that will mean that the current function will not work on your current branch unless you rebase.